vendor/modules.txt

```diff
@@ -81,7 +81,6 @@ github.com/Microsoft/hcsshim/osversion
 # github.com/NYTimes/gziphandler v1.1.1 => github.com/NYTimes/gziphandler v1.1.1
 github.com/NYTimes/gziphandler
 # github.com/PuerkitoBio/purell v1.1.1 => github.com/PuerkitoBio/purell v1.1.1
-## explicit
 github.com/PuerkitoBio/purell
```

Comment:
We can't actually get rid of it yet, right?  I still see one dep on it (vendor/github.com/go-openapi/jsonreference/reference.go).  Should we port this same change to go-openapi ?

Reply:
looks like it asks for a more demanding normalization from the library:
https://github.com/go-openapi/jsonreference/blob/master/reference.go#L117

https://github.com/PuerkitoBio/purell/blob/master/README.md

> FlagsSafe NormalizationFlags = FlagLowercaseHost | FlagLowercaseScheme | FlagUppercaseEscapes | FlagDecodeUnnecessaryEscapes | FlagEncodeNecessaryEscapes | FlagRemoveDefaultPort | FlagRemoveEmptyQuerySeparator

so i don't think the replacement function in this PR is sufficient, as it only handles the duplicate slashes.